GitUI/UserControls/RevisionGridClasses/DvcsGraph.Designer.cs

```diff
@@ -2,9 +2,6 @@
 {
     partial class DvcsGraph
     {
-        /// <summary> 
```

Comment:
❗️ I need to figure out what happened here. This is a problem. We can't be requiring users to manually edit designer-generated files.

Reply:
I think this was a legitimate user-created error. However yes you may like to exclude `*.Designer.cs` etc. I believe the WinForms designer won't delete code developers add to these files though. I don't see this as a deficiency in the analyser, from what I know.